download/oci_download.go

```diff
@@ -271,7 +271,7 @@ func (d *OCIDownloader) download(ctx context.Context, m metrics.Metrics) (*downl
 		return nil, err
 	}
 	loader := bundle.NewTarballLoaderWithBaseURL(fileReader, d.localStorePath)
-	reader := bundle.NewCustomReader(loader).WithBaseDir(d.localStorePath).
+	reader := bundle.NewCustomReader(loader).
```

Comment:
Wouldn't this break verification for some existing OCI bundles?

Reply:
As there was no test for it I assume that it didn't work before. 

If you want to try that just add revert the line you mentioned and rerun the test.

The test uses a bundle that is signed directly via opa:

https://github.com/open-policy-agent/opa/blob/c66511f9068ceafa6e3a639a0a99e5dda541de05/download/oci_download_test.go#L21

I still consider the https://github.com/open-policy-agent/opa/pull/6147 the more correct thing to do, although I see that might lead to not backward compatible changes. As my assumption is that I could sign a bundle in one folder for example: `/home/userx/bundles/myBundle` then copy that to another system into `/data/bundles/myBundle` and verify the signature without having to move to bundle to a specific folder. Besides that point the other usages of the bundle reader do not seem to us the `WithBaseDir` option.

https://github.com/open-policy-agent/opa/blob/de896d96f52aa041f9f9966851270e70db3b6816/download/download.go#L331-L338

or 

https://github.com/open-policy-agent/opa/blob/de896d96f52aa041f9f9966851270e70db3b6816/internal/bundle/utils.go#L101-L110